modules/user/server-ts/sql.js

```diff
@@ -163,13 +161,23 @@ class User {
     return returnId(knex('auth_linkedin')).insert({ ln_id: id, display_name: displayName, user_id: userId });
   }
 
-  async editUser({ id, username, email, role, isActive, password }) {
+  editUser({ id, username, email, role, isActive, password }) {
     let localAuthInput = { email };
     if (password) {
-      const passwordHash = await bcrypt.hash(password, 12);
-      localAuthInput = { email, password_hash: passwordHash };
+      console.log('editUser password');
```

Comment:
console.log shouldn't be in the final version

Reply:
fixed